internal/resolvers/imgur/check.go

```diff
@@ -0,0 +1,14 @@
+package imgur
+
+import (
+	"net/url"
+	"strings"
+)
+
+func check(url *url.URL) bool {
+	// TODO: Make a shared helper function that does basically this.
+	// helpers.IsSubdomainOf(url.Host, "imgur.com")
+	isImgur := strings.HasSuffix(url.Host, ".imgur.com") || url.Host == "imgur.com"
```

Comment:
Are we just straight up accepting a wildcard for `*.imgur.com`? How about domains like `https://api.imgur.com/` should we also support them?

Reply:
This is handled in imgur/load.go:40 where we return the `ErrDontHandle` error to signify that the generic resolver should step in whenever a non-image URL has been handled